app/models/automated_checks/claim_verifiers/census_subjects_taught.rb

```diff
@@ -41,7 +41,7 @@ def no_data
       end
 
       def no_match
-        return if school_workforce_census.nil? || eligible?
+        return unless school_workforce_census.nil? || !eligible?
```

Comment:
Just double-checking with you this is as expected, at a glance `school_workforce_census.nil?` isn't negated or the condition isn't `&&`.